src/styles/components/_docs.scss

```diff
@@ -14,6 +14,15 @@
     min-height: calc(100vh - #{$mobileHeaderH});
     padding: 30px 15px;
   }
+
+  select {
+    appearance: none;
+    outline: 0; box-shadow: none; border: 0 !important; border-radius: .25em;
```

Comment:
Not use `!important` please.

`.foo .bar select { border: 0; }` instead of `select { border: 0 !important; }`

It is better to use a greater depth to override a CSS property.

Because the important thing is very complicated to overload.

Reply:
You have to use the `em` only to the font face.